modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -12,15 +12,14 @@ class IntentToFileController < ClaimsApi::V2::ApplicationController
         #
         # @return [JSON] ITF record
         def type
-          itf_types = ClaimsApi::V2::IntentToFile::ITF_TYPES_TO_BGS_TYPES.keys
-          type = request.path_parameters[:type].downcase
-          unless itf_types.include?(type)
+          type = params[:type].downcase
```

Comment:
we were using `params` in some spots and `request.path_parameters` in others.. for consistency went back to `params`